lib/codegen_types.re

```diff
@@ -15,7 +15,8 @@ let wrap_ident= (ident:string) : string => {
 }
 let codegen_atom = (atom: B.atom): string => {
    switch(atom) {
-   | B.TOKEN => "string" /* tokens are string */
+   | B.TOKEN(None) => "string" /* tokens are empty */
+   | B.TOKEN(Some(name)) => wrap_ident(name)
```

Comment:
so, this is wrong. it should be something like "token" that we could define to be
a synonymous for "string" in the generated type file.